src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,17 @@
+@mixin square ($size) {
```

Comment:
Add a comment explaining the purpose of the `square` mixin. For example: `// Creates a square shape with given size and centers it 🖼️`.